inc/enqueue.php

```diff
@@ -67,11 +67,11 @@ function uds_wp_metaboxes_scripts() {
 		global $pagenow;
 		$the_theme     = wp_get_theme();
 		$theme_version = $the_theme->get( 'Version' );
-		$js_metaboxes_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/metaboxes.js' );
+		$js_admin_blocks_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/admin-blocks.js' );
 		$css_metaboxes_version = $theme_version . '.' . filemtime( get_template_directory() . '/css/admin.min.css' );
 		if ( 'post.php' == $pagenow || 'post-new.php' == $pagenow ) {
 
-			wp_enqueue_script( 'uds-wordpress-sidebar-metabox-scripts', get_template_directory_uri() . '/js/metaboxes.js', array( 'jquery' ), $js_metaboxes_version, true );
+			wp_enqueue_script( 'uds-wordpress-sidebar-metabox-scripts', get_template_directory_uri() . '/js/admin-blocks.js', array( 'jquery' ), $js_admin_blocks_version, true );
 			wp_enqueue_style( 'uds-wordpress-sidebar-metabox-styles', get_template_directory_uri() . '/css/admin.min.css', array(), $css_metaboxes_version );
```

Comment:
Just a reminder to use the same script handle from line 74 above, once we have changed it to better match what this script actually does.

Reply:
This css file only has the metabox related styles, so I'm not sure if we need to rename it or to just delete the file since it's not being used at the time.
What do you think?